examples/jsm/nodes/lighting/LightsNode.js

```diff
@@ -120,8 +120,8 @@ export const lightsWithoutWrap = nodeProxy( LightsNode );
 export function addLightNode( lightClass, lightNodeClass ) {
 
 	if ( LightNodes.has( lightClass ) ) throw new Error( `Redefinition of light node ${ lightNodeClass.name }` );
```

Comment:
`lightNodeClass.type`?

Reply:
I'm referring to updating here, if the code is minified it will preserve the name.
`Redefinition of light node ${ lightNodeClass.type }`

Reply:
I've changed `name` to `type` in both cases. (This also required swapping the order of `addNodeClass` and `addLightNode` statements so that `type` is set already when executing `addLightNode`)